desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,24 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### Upgrades
+
+- [Buildx 0.8.1](https://github.com/docker/buildx/releases/tag/v0.8.1)
```

Comment:
Should we be explaining what this fixes? I haven’t looked at the changes to be honest.

Reply:
I checked a bit over release notes and we just put the upstream tag usually